### PR TITLE
Support attachments in Mandrill adapter

### PR DESCRIPTION
**Pull request by princemaple**
Docs ref: https://mandrillapp.com/api/docs/messages.JSON.html#method=send suggests

I don't use mandrill either, can't test...

### CODE REVIEW

**Comment by stevedomin** [comment]
Trying now

File: lib/swoosh/adapters/mandrill.ex [hunk 1]
```diff
@@ -90,11 +91,20 @@ defmodule Swoosh.Adapters.Mandrill do
   defp prepare_bcc(body, %Email{bcc: []}), do: body
   defp prepare_bcc(body, %Email{bcc: bcc}), do: prepare_recipients(body, bcc, "bcc")
 
+  defp prepare_attachments(body, %Email{attachments: []}), do: body
+  defp prepare_attachments(body, %Email{attachments: attachments}) do
+    Map.put(body, "attachments", Enum.map(attachments, &%{
+      "name" => &1.filename,
+      "type" => &1.content_type,
+      "content" => &1 |> File.read! |> Base.encode64
```
Comment (stevedomin) [inline lib/swoosh/adapters/mandrill.ex]: Shouldn't this be `&1.path |> File.read! |> Base.encode64`?

**Comment by stevedomin** [comment]
I just realised I can't try the Mandrill adapter because Mandrill has no free plan :/

File: lib/swoosh/adapters/mandrill.ex (hunk 1 above)
Comment (princemaple) [inline lib/swoosh/adapters/mandrill.ex]: My bad.

**Pull request merged**